app/jmeter/jira.jmx

```diff
@@ -3061,12 +3061,12 @@ if ( sleep_time &gt; 0 ) {
                 <hashTree/>
                 <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
                   <collectionProp name="Asserion.test_strings">
-                    <stringProp name="1182867844">[&quot;project-key&quot;]=&quot;\&quot;${project_key}\&quot;&quot;;</stringProp>
+                    <stringProp name="-1259658522">.*project-key.*${project_key}</stringProp>
```

Comment:
What is the name here? Would it be possible to have a meaningful name here or is this an autogenerated thing?

Reply:
It is auto-generated name for pattern in assetion